packages/paste-core/components/base-radio-checkbox/src/BaseRadioCheckbox.tsx

```diff
@@ -13,17 +13,18 @@ import type {BoxProps} from '@twilio-paste/box';
 export interface BaseRadioCheckboxControlProps extends SiblingBoxProps, Pick<BoxProps, 'element'> {
   children: NonNullable<React.ReactNode>;
   disabled?: boolean;
+  applyHoverStyles?: boolean;
 }
 const BaseRadioCheckboxControl = React.forwardRef<HTMLSpanElement, BaseRadioCheckboxControlProps>(
-  ({children, disabled, element = 'BASE_RADIO_CHECKBOX_CONTROL', ...props}, ref) => {
+  ({children, disabled, applyHoverStyles = false, element = 'BASE_RADIO_CHECKBOX_CONTROL', ...props}, ref) => {
     return (
       <SiblingBox
         as="span"
         ref={ref}
         display="flex"
         element={element}
         backgroundColor="colorBackgroundBody"
-        borderColor="colorBorder"
+        borderColor={applyHoverStyles ? 'colorBorderPrimaryStronger' : 'colorBorder'}
         borderStyle="solid"
         borderWidth="borderWidth10"
         height="sizeSquare50"
```

Comment:
(this comment should be on line 41ish)

todo: I think we might have missed this in the One Twilio updates reviews, but on focus, the border color should stay at colorBorder (instead of colorBorderPrimaryStronger)